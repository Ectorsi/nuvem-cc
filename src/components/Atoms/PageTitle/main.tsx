import * as React from 'react';

type PageTitleProps = {
    title: string,
};

const PageTitle = ({ title }: PageTitleProps) => {
    return (
        <h1>{title}</h1>
    )
};

export default PageTitle;