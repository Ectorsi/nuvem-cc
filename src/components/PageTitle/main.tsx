import * as React from 'react';

const PageTitle = () => {
    return (
        <h1>Title</h1>
    )
};

export default PageTitle;