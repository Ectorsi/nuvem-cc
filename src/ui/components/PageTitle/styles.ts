import styled from 'styled-components';
import Theme from '../../../styles/theme';

export const Container = styled.h1`
    margin-bottom: 10px;
    color: ${Theme.colors.font.secondary};
`;
