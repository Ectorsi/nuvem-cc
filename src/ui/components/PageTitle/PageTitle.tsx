import * as React from 'react';
import * as S from './styles';

type PageTitleProps = {
    title: string,
};

const PageTitle = ({ title }: PageTitleProps) => {
    return (
        <S.Container>
            {title}
        </S.Container>
    )
};

export default PageTitle;