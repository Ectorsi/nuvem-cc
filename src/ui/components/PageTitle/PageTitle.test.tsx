import { render, screen } from '@testing-library/react';

import PageTitle from './PageTitle';

describe('PageTitle unit test', () => {
    it('should be able to show the page title', () => {
        const { container } = render(<PageTitle title="Test" />);
        const title = screen.getByText(/Test/i);
        expect(title).toBeInTheDocument();
        expect(container.firstChild).toMatchSnapshot();
    });
});
