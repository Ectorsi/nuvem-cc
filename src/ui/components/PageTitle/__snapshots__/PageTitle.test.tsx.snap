// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle unit test should be able to show the page title 1`] = `
<h1
  class="sc-beqWaB ekCqDZ"
>
  Test
</h1>
`;
